### ISSUE TITLE
Differentiate changed line icon for whitespace only changes

### ISSUE THREAD

**Issue by etler** [jisaacks/GitGutter#260]
When reformatting files, it can be a little confusing figuring out what lines have significant changes when there are major whitespace changes. Having a different icon for whitespace only changes would be very helpful in those cases.

Is this a feature that would be worth supporting in GitGutter? Also, what should the icon be? An unfilled outline of a square would probably be the most obvious.

**Comment by jisaacks** [comment]
@etler, I don't think this is something that I care to work on myself, I rarely make significantly white space only changes enough to ever care. However, if someone proposed a clean solution that could be off by default and turned on via settings, I would probably merge it.

**Comment by etler** [comment]
Cool, I think I'll take a stab at it. I just wanted to check if there was interest in it so my effort wouldn't go to waste.

**Comment by jisaacks** [comment]
Also, before you spend any effort, it might be worth checking to see if [this setting](https://github.com/jisaacks/GitGutter/blob/master/GitGutter.sublime-settings#L33) helps you.

**Comment by etler** [comment]
Nice, thanks for pointing that out!
